pkg/api/v1alpha1/cloudstackdatacenterconfig_types.go

```diff
@@ -177,7 +177,7 @@ func (v *CloudStackDatacenterConfig) SetDefaults() {
 				Account:               v.Spec.Account,
 				Domain:                v.Spec.Domain,
 				ManagementApiEndpoint: v.Spec.ManagementApiEndpoint,
-				CredentialsRef:        "Global",
+				CredentialsRef:        "global",
```

Comment:
let's also resolve the TODO above in the Validate() method. I think we can call Validate() from cloudstack.go or validator.go